src/styles/card.scss

```diff
@@ -0,0 +1,95 @@
+$first-color: #060b35;
+$second-color: #616070;
+$button-color: #00acdc;
+
+.card {
+  border: 1px solid #f3f3f3;
```

Comment:
separate file for this bem block

Reply:
> separate file for this bem block

I didn't quite understand what to do. The card block is already in a separate card.scss file.